### PR TITLE
Creates character list

### PR DESCRIPTION
**Pull request by mellymillionz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Tale of the Detached HEAD
 ### A spooky exercise in version control
 
+Character List:
```
Comment (mjprude) [inline README.md]: Let's make this `Character List:` a heading

**Review by mjprude — CHANGES_REQUESTED** [review]
Make things more markdown-y please

**Review by mjprude — APPROVED** [review]
LGTM

**Review by tedwardmah — APPROVED** [review]
Trusting @mjprude on this one, looks good!

**Pull request merged**